.github/workflows/conformance-clustermesh.yaml

```diff
@@ -126,6 +126,7 @@ jobs:
             cm-auth-mode-2: 'legacy'
             maxConnectedClusters: '255'
             ciliumEndpointSlice: 'enabled'
+            policy-default-local-cluster: true
```

Comment:
Nit. rather than enabling it for the first 5 entries and disabling for the last, could you please alternate enabled/disabled a bit more? Just to increase the possibilities of covering multiple combinations.